kv/variables.go

```diff
@@ -18,6 +18,9 @@ type Variables struct {
 	// BackoffLockFast specifies the LockFast backoff base duration in milliseconds.
 	BackoffLockFast int
 
+	// BackOffWeight specifies the weight of the max back off time duration.
+	BackOffWeight int
```

Comment:
backoff weight is a little confusion, how about directly set the max backoff time?

Reply:
There are many kinds of backoffs. Maybe set them separately in the future.